test/test_basic.py

```diff
@@ -638,6 +638,7 @@ def test_with_local_changes(
     dbsync_status(mc)
 
 
+@pytest.mark.skip(reason="Currently being reworked in MM")
```

Comment:
why does this need to be marked as skipped?

Reply:
The test was failing when runned localy, I thought it might be caused by changes in mergin-client. But it seems to be working now, so maybe it was related to project marked for deletion from previous test runs... anyways the skip is removed.